job_service/api/job_api.py

```diff
@@ -19,24 +17,24 @@
 
 
 @job_api.get("/jobs")
-@validate()
-def get_jobs(query: GetJobRequest):
-    logger.debug(f"GET /jobs with query: {query}")
-    jobs = job_db.get_jobs(query)
+def get_jobs():
+    query_validated = GetJobRequest(**request.json)
```

Comment:
This is named "query" which means it originally gets the fields from the query string and not the json payload, so this will break the current API. (And is the reason the test needed to be updated below)